src/EnergyPlus/GroundTemperatureModeling/BaseGroundTemperatureModel.hh

```diff
@@ -51,7 +51,6 @@
 // EnergyPlus Headers
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/EnergyPlus.hh>
-#include <EnergyPlus/GroundTemperatureModeling/GroundTemperatureModelManager.hh>
 
```

Comment:
Defect file? Unit test?

Reply:
What does the compiler do with this recirsive code?